test/addons/rbd-mirror/vrc-flatten-sample.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: replication.storage.openshift.io/v1alpha1
+kind: VolumeReplicationClass
+metadata:
+  name: vrc-sample
```

Comment:
Name is duplicate of existing VRClass and hence will overwrite the default (based on order of apply called).